rpm/qtdeclarative.spec

```diff
@@ -267,6 +284,8 @@ touch .git
 %endif
 
 make %{?_smp_mflags}
+make qtdeclarative_docs
```

Comment:
surround this with a:

    %if 0%{?qt5_release_build}
    %endif

and it should do the trick. that macro ought to be set for development builds all the time :)

(YMMV, I'm getting the syntax off the top of my head)